Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -126,25 +143,96 @@
     OTHER_MARK_ALL_AS_READ_INDEX,
     OTHER_CLEAR_CACHE_INDEX,
     OTHER_REPORT_BUG_INDEX,
-    OTHER_COUNT
 };
 
 enum
 {
     LABS_USE_JITSI_WIDGET_INDEX = 0,
-    LABS_COUNT
 };
 
 enum
 {
     SECURITY_BUTTON_INDEX = 0,
-    SECURITY_COUNT
 };
 
-#define SECTION_TITLE_PADDING_WHEN_HIDDEN 0.01f
-
 typedef void (^blockSettingsViewController_onReadyToDestroy)(void);
 
+#pragma mark - Row
```

Comment:
Can we move them to 2 dedicated classes, please?

Reply:
👍